packages/checkbox/src/Checkbox/Checkbox.tsx

```diff
@@ -22,36 +29,39 @@ import {
   labelStyle,
   labelTextStyle,
 } from './Checkbox.style';
-import { CheckboxProps } from './types';
+import { CheckboxProps } from './Checkbox.types';
 
 /**
  * Checkboxes should be used whenever a user has an option they’d like to opt in or out of.
  *
  * Unlike toggles, checkboxes are used for actions, or features, that don’t immediately turn on or off. Checkboxes are usually found in forms as opposed to config pages.
  */
 function Checkbox({
-  darkMode: darkModeProp,
+  animate = true,
```

Comment:
Do you wanna organize by defaults and aliases?

Reply:
i did it alphabetically but i can revert

Reply:
I have no preference; whatever floats ur boat